docs/source/preview_datasets.md

```diff
@@ -1,84 +1,62 @@
-# Preview data in Kedro-Viz
+# Preview datasets in Kedro-Viz
 
-This page describes how to preview data from different datasets in a Kedro project with Kedro-Viz. Dataset preview was introduced in Kedro-Viz version 6.3.0, which offers preview for `CSVDatasets` and `ExcelDatasets`.
+To provide users with a glimpse of their datasets within a Kedro project, Kedro-Viz offers a preview feature. This feature was introduced in Kedro-Viz version 6.3.0 and expanded upon in version 8.0.0. Initially, it supported `CSVDataset` and `ExcelDataset`, and was later extended to encompass additional dataset types such as `PlotlyDataset` and image datasets like `MatplotlibWriter`.
 
-We use the {doc}`spaceflights tutorial<kedro:tutorial/spaceflights_tutorial>` to demonstrate how to add data preview for the `customer`, `shuttle` and `reviews` datasets. Even if you have not yet worked through the tutorial, you can still follow this example; you'll need to use the Kedro starter for the spaceflights tutorial to generate a copy of the project with working code in place.
 
-If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+Currently, Kedro-Viz supports four types of previews:
 
-```{important}
-We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.0). To check the version installed, type `kedro -V` in your terminal window.
-```
+1. **TablePreview:** For datasets returning tables/dataframes.
+2. **JSONPreview:** For datasets returning JSON objects.
+3. **PlotlyPreview:** For datasets returning Plotly JSON objects.
+4. **ImagePreview:** For datasets returning base64-encoded image strings.
 
-In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+While we currently support the aforementioned datasets, we are soon going to extend this functionality to include other datasets. Users with custom datasets can also expand the preview functionality, , which is covered in the section [Extend Preview to Custom Datasets](./preview_custom_datasets.md).
```

Comment:
Do we have Vale enabled on this repo? I get the sense that _aforementioned_ would be flagged as too wordy

Reply:
i don't think so.